### DISCUSSION TITLE
Readme needs more ELI5?

### DISCUSSION THREAD

**Discussion by stianhoiland** [orbitalquark/scinterm#23] [General]
This looks like a cool project! But, and even though I read the readme, I am a little unsure what it actually is.

AFAIK, Scintilla is a text/code editing *library* and doesn't do anything by itself. So, is scinterm also a library? Or is scinterm more like SciTE (a minimal text/code editor built for showcasing the Scintilla library) for curses?

Maybe you could update the readme for doofuses like myself :)

**Comment by rhaberkorn** [comment]
Scintilla includes widgets for a few popular widget toolkits (Gtk, Qt, Windows GDI, Apple Cocoa). Scinterm adds support for a Curses "widget". As afar as I recall, there was discussion to integrate Scinterm as a permanent port into the main Scintilla repository. But you have to ask the maintainer why this didn't work out.

**Comment by stianhoiland** [comment]
Aha, I understand now. I guess the missing piece for me was that Scintilla supplies special support for select widget toolkits—so scinterm is that but for curses. Thank you for clarifying.
